src/GZCTF/Resources/Program.zh-CN.resx

```diff
@@ -920,6 +920,9 @@
   <data name="Challenge_UseAssetsApiForDynamic" xml:space="preserve">
     <value>动态附件题目请使用 assets API 上传附件</value>
   </data>
+  <data name="Challenge_SubmissionLimitExceeded" xml:space="preserve">
```

Comment:
You don't need to create strings for languages other than English as they are auto-generated by an external workflow.